calcom/cal.com
# feat: org owner-admin cant see the teams created by other owner-admin 9982 cal 2120

## What does this PR do?

- If user is owner/admin in org in settings tabs it should displays teams that other owner/admin created but current user don't belong to.

Fixes #9982 

## Requirement/Documentation



- If there is a requirement document, please, share it here.
- If there is ab UI/UX design document, please, share it here.

## Type of change




## How should this be tested?



- Are there environment variables that should be set? No
- What are the minimal test data to have? Org teams with two owner/admins
- What is expected (happy path) to have (input and output)? One of the two owner/admins should create a new team excluding the other one and from the excluded user he should be able to see other teams created in org.
- Any other important info that could help to test that PR

## Mandatory Tasks


## Checklist



- I haven't added tests that prove my fix is effective or that my feature works
